guides/common/attributes-base.adoc

```diff
@@ -17,10 +17,10 @@
 :RepoRHEL7ServerOptional: rhel-7-server-optional-rpms
 :RepoRHEL7ServerAnsible: rhel-7-server-ansible-2.9-rpms
 // For Beta, change to "Beta". For GA releases, change to, for example, "6.8".
-:RepoRHEL7ServerSatelliteServerProductVersion: rhel-server-7-satellite-6-rpms
+:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-6.10-rpms
 :RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-server-7-satellite-6.9-rpms
-:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-server-7-satellite-capsule-6-rpms
-:RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-tools-6-rpms
+:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-7-server-satellite-capsule-6.10-rpms
+:RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-tools-6.10-rpms
 :RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-6-rpms
 // Do not update the puppet4 repo versions. They must stay at 6.3.
 :RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
```

Comment:
The last time I tested this:

```
subscription-manager repos --enable=rhel-7-server-rpms \
--enable=rhel-server-7-satellite-6-beta-rpms \
--enable=rhel-7-server-satellite-maintenance-6-beta-rpms \
--enable=rhel-server-rhscl-7-rpms \
--enable=rhel-7-server-ansible-2.9-rpms
```

That was beta, let me verify this.

Reply:
Checking:

* rhel-7-server-satellite-6.10-rpms - https://access.redhat.com/downloads/content/satellite/6.10.0-3.el7sat/noarch/fd431d51/package
* 15:10 <Zhenech> for capsule its  rhel-7-server-satellite-capsule-6.10-rpms
* 15:11 <Zhenech> and for tools its  rhel-7-server-satellite-tools-6.10-rpms

* rhel-7-server-satellite-capsule-6.1-rpms - https://access.redhat.com/downloads/content/capsule-installer/2.3.25-1.el7sat/noarch/fd431d51/package
* 

rhel-7-server-satellite-capsule-6.1-rpms
--




rhel-7-server-satellite-capsule-6.1-rpms
--




rhel-7-server-satellite-capsule-6.1-rpms
--


Okay thanks!

Reply:
your IRC-to-GH-paste capabilities need more :beers: